code/modules/jobs/job_types/roguetown/yeomen/longshoreman.dm

```diff
@@ -0,0 +1,60 @@
+/datum/job/roguetown/longshoreman
+	title = "Longshoreman"
+	flag = LONGSHOREMAN
+	department_flag = GARRISON
+	faction = "Station"
+	total_positions = 4
+	spawn_positions = 4
+	selection_color = JCOLOR_SOLDIER
+	allowed_sexes = list(MALE, FEMALE)
+	allowed_races = list(/datum/species/akula, /datum/species/human/northern, /datum/species/lizardfolk)
```

Comment:
Why are longshoremen race-locked? Odd for a commoner role. Is this just because of clothing sprites?